content/riak/ts/1.6.0/using/querying/select/aggregate-functions.md

```diff
@@ -143,3 +146,29 @@ Returns `NULL` if no values were returned or all values were `NULL`.
 |-------------------|-------------|
 | sint64            | sint64 |
 | double            | double |
+
+### `PERCENTILE_DISC`, `PERCENTILE_CONT`
+
+Calculate a percentile, given as a value in the range [0..1], in a population of entries in the selection, with null values discarded.  The column argument must be of numeric type (`sint64`, `double` or `timestamp`).  Return type is the type of the column argument for `PERCENTILE_DISC`, and `double` for `PERCENTILE_CONT`.
```

Comment:
I got confused about the purpose of the range. I was thinking that would be the output, not the input.